src/pronouns/web.clj

```diff
@@ -28,10 +28,14 @@
      :headers {"Content-Type" "text/css"}
      :body (slurp (io/resource "pronouns.css"))})
 
-  (GET "/*" {params :params}
-       {:status 200
-        :headers {"Content-Type" "text/html"}
-        :body (pages/pronouns (:* params) pronouns-table)})
+  (GET "/*" {params :params headers :headers}
+       (if (= "application/json" (.toLowerCase (get headers "accept")))
+         {:status 200
+          :headers {"Content-Type" "application/json"}
+          :body (pages/pronouns (:* params) pronouns-table :json)}
```

Comment:
can we just dispatch to two separate functions here, instead of passing that last parameter?